src/js/wall/B2cLogin.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { connect } from "react-redux";
+import styled from "styled-components";
+import { loginSucceeded } from "../account/actions";
+import { getLoginRequest } from "../app/authConfig";
+import { getColor } from "../app/theme";
+import { Button, Divider } from "../base";
+import { WallSubheader } from "./Container";
+
+const StyledB2cLogin = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: stretch;
+    margin: 10px 0px;
+    padding: 0px 15px;
+`;
+
+const BetaTag = styled.span`
+    color: ${props => getColor({ color: "greyDark", theme: props.theme })};
+    margin-left: auto;
+    transform-text: uppercase;
+`;
+
+const StyledDivider = styled(Divider)`
+    margin-bottom: 30px;
+`;
+
+export const B2cLogin = ({ setLogin }) => (
```

Comment:
I may be fuzzy on this, but all Virtool callbacks in props are named like `onSetLogin`. This makes it easy to separate functions from other variables in component definitions.

Pretty sure it is React convention as well.